tests/cluster_policies/__init__.py

```diff
@@ -89,8 +89,8 @@ class TimedOperator(BaseOperator, ABC):
 def task_policy(task: TimedOperator):
     if task.task_type == 'HivePartitionSensor':
         task.queue = "sensor_queue"
-    if task.timeout > timedelta(hours=48):
```

Comment:
This is relative to the `TimedOperator` just above this policy though. The `[START example_task_cluster_policy]` marker was moved above the `TimedOperator` declaration in #23468 so it will show up in the Cluster Policy doc when published.  Should the `TimedOperator` piece be removed altogether then?

Reply:
It seems complex to change the whole example. everything I touch produce another error.
I'll leave it to someone more experienced with cluster policy.